### PR TITLE
+ search user, document

### PR DESCRIPTION
**Pull request by giapnguyenitf**

### CODE REVIEW

**Comment by giapnguyenitf** [comment]
ready

File: app/Http/Controllers/SearchController.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{
+    public function index(Request $request)
+    {
+        $keyword = $request->search;
+        $categories = Category::with('subCategories')->where([ 
+            ['parent_id', '=', config('setting.null')],
+            ['status', '=', config('setting.true')],
+        ])->orderBy('name', 'asc')->get();
+
+        $documents = Document::where('name', 'like', '%'.$keyword.'%')
```
Comment (trangkarol) [inline app/Http/Controllers/SearchController.php]: Convention

File: app/Http/Controllers/SearchController.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            ->orWhere('description', 'like', '%'.$keyword.'%')
+            ->orWhereIn('user_id', function ($query) use ($keyword) {
+                $query->select('id')->from('users')->where('name', 'like', '%'.$keyword.'%');
```
Comment (trangkarol) [inline app/Http/Controllers/SearchController.php]: Convention

File: app/Http/Controllers/SearchController.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    public function index(Request $request)
+    {
+        $keyword = $request->search;
+        $categories = Category::with('subCategories')->where([ 
+            ['parent_id', '=', config('setting.null')],
+            ['status', '=', config('setting.true')],
+        ])->orderBy('name', 'asc')->get();
+
+        $documents = Document::where('name', 'like', '%'.$keyword.'%')
+            ->orWhere('description', 'like', '%'.$keyword.'%')
```
Comment (trangkarol) [inline app/Http/Controllers/SearchController.php]: Convention

**Comment by trangkarol** [comment]
commented

**Comment by giapnguyenitf** [comment]
ready

**Comment by trangkarol** [comment]
merged

**Pull request merged**